src/Neo/SmartContract/Native/RoleManagement.cs

```diff
@@ -26,7 +26,16 @@ public sealed class RoleManagement : NativeContract
     {
         [ContractEvent(0, name: "Designation",
             "Role", ContractParameterType.Integer,
-            "BlockIndex", ContractParameterType.Integer)]
+            "BlockIndex", ContractParameterType.Integer,
+            Hardfork.HF_Domovoi)]
```

Comment:
The next one supposed to be HF_Echidna, may be borrowed from https://github.com/neo-project/neo/pull/3175/commits/4d5cee903fa77965d590a131243136b227c75c4b.